test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -1235,19 +1191,14 @@ INSTANTIATE_TEST_CASE_P(
     // mkModelTc(fc_softmax_mse_distribute_validate, "3:1:5:3", 1),
     // mkModelTc(fc_softmax_cross_distribute_validate, "3:1:5:3", 1),
     // mkModelTc(fc_sigmoid_cross_distribute_validate, "3:1:5:3", 1)
-    mkModelTc(rnn_basic, "1:1:1:1", 1),
-    mkModelTc(rnn_return_sequences, "1:1:2:1", 1),
-    mkModelTc(lstm_basic, "1:1:1:1", 1),
-    mkModelTc(lstm_return_sequence, "1:1:2:1", 1),
-    mkModelTc(lstm_return_sequence_with_batch, "2:1:2:1", 1),
-    mkModelTc(lstm_return_sequence_with_batch_n, "2:1:2:1", 2),
-    mkModelTc(multi_lstm_return_sequence, "1:1:1:1", 1),
-    mkModelTc(multi_lstm_return_sequence_with_batch_n, "2:1:1:1", 2),
```

Comment:
I wonder why remove negative testcases from here?

Reply:
here _n is not negative case but denotes n-iteration.

n-iteration is not needed as the patch makes all the test have 10 iterations.